frontend/src/RarityModule/CreateNewRarity/components/CreateNewRarity.vue

```diff
@@ -84,10 +89,11 @@ const onSubmit = () => {
 };
 
 const fetchGames = () => {
```

Comment:
if you don't mind the extra work, can you move these kind of functions into their own module's, and you create a folder called api, and create a file, for example this case, it will be games.ts, and there is this function that is taking whatever is needed as the payload for their own requests, and it returns whatever from await callApi directly.